packages/fluentui/react-northstar/src/components/Carousel/CarouselPaddlesContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {
+  useFluentContext,
+  useTelemetry,
+  useStyles,
+  useAccessibility,
+  getElementType,
+  useUnhandledProps,
+  ComponentWithAs,
+} from '@fluentui/react-bindings';
+import {
+  commonPropTypes,
+  UIComponentProps,
+  ContentComponentProps,
+  ChildrenComponentProps,
+  childrenExist,
+} from '../../utils';
+import { FluentComponentStaticProps } from '../../types';
+import { Accessibility } from '@fluentui/accessibility';
+
+export interface CarouselPaddlesContainerProps extends UIComponentProps, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+
+export type CarouselPaddlesContainerStylesProps = never;
+export const carouselPaddlesContainerClassName = 'ui-carrouselpaddles_container';
```

Comment:
Can you add a before/after of the DOM structure ? Just to communicate that that the two buttons are no longer direct children of the carousel container

Reply:
There wouldn't be visual changes so not sure if I get the point for having screenshot

Reply:
they wouldn't be visual changes, but can definitely raise questions why applied styles might suddenly be different or why E2E tests fail because of a selector that relies on only one div container before the button